submissions/rubyAdeniregun/todo-app/src/components/TodoItem.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'; 
+import '../App.css';
+import './style.css';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSquare } from '@fortawesome/fontawesome-free-regular'
+import { faCheckSquare } from '@fortawesome/fontawesome-free-regular'
+
+
+class TodoItem extends React.Component {
+
+  render () {
+
+    const {todo} = this.props;
+
+    return (
+      <div className={'todoItem' + (todo.completed ? ' completed' : '')} onClick={this.toggleTodo}>
+        {todo.text} 
+        
+        <div className="box">
+        <FontAwesomeIcon icon={todo.completed ? faCheckSquare : faSquare}/>
```

Comment:
Instead of this have a check box, just like normal html and add a class and add attribute content to it, style it to the fit the checkbox @rubyade